src/readme/rules.ts

```diff
@@ -572,13 +587,14 @@ const rules: IRule[] = [
   {
     available: false,
     eslintRule: 'no-alert',
-    tslintRule: 'no-alert',
+    tslintRule: 'ban',
     category: 'Best Practices',
     description: 'disallow the use of `alert`, `confirm`, and `prompt`',
     eslintUrl: 'http://eslint.org/docs/rules/no-alert',
-    provider: 'tslint-eslint-rules',
+    tslintUrl: 'https://palantir.github.io/tslint/rules/ban',
+    provider: 'native',
     usage: `~~~json
-    "no-alert": true
+    "ban": [true, ['alert']]
```

Comment:
and switch `available` to true since tslint provides it. In this case I would expand on the description to explain how to use the use. See `no-unused-vars`. This rule became depracated but I put a link and an explanation on its description.